dotenv/mod.ts

```diff
@@ -1,187 +1,106 @@
 // Copyright 2018-2022 the Deno authors. All rights reserved. MIT license.
+// This module is browser compatible.
 
-import { difference, removeEmptyValues } from "./util.ts";
-
-export interface DotenvConfig {
-  [key: string]: string;
+export interface DotEnvObject {
+  env: Record<string, string>;
+  exports: string[];
 }
 
-export interface ConfigOptions {
-  path?: string;
-  export?: boolean;
-  safe?: boolean;
-  example?: string;
-  allowEmptyValues?: boolean;
-  defaults?: string;
+// Deno should expose a type for Deno.env
+// Ideally Map-like as discussed in https://github.com/denoland/deno/issues/10349
```

Comment:
Using Deno NS types will prevent type-safe browser compatibility. It is useful for these types to be declared independently.